src/NHSD.BuyingCatalogue.Solutions.API/QueryModels/SupplierSearchQuery.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NHSD.BuyingCatalogue.Solutions.Contracts;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.QueryModels
+{
+    public sealed class SupplierSearchQuery
+    {
+        private string _solutionPublicationStatus;
+
+        public string Name { get; set; }
+
+        public string SolutionPublicationStatus
```

Comment:
My preference would be for it to remain exposed (and to remove the flag) as I think that taking a query parameter that accepts the publication status is probably 'more RESTful' and the better longer-term design.

Flag arguments, in most cases, are a tad smelly. The associated tests were also simpler before the flag argument was agreed. The flag argument is really just a kludge to keep the FE happy, in my mind. I think the API would be better without it.

Therefore, I would prefer it if the FE were sending the publication status rather than the flag.

Reply:
After discussion I think we are now in agreement & understand each others points. Action on this one then is to remove the "limit" flag & use the `SolutionPublicationStatus`. Will let the FE team know & update the JSON contract.